<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradingManagerClass</class>
 <widget class="QWidget" name="GradingManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GradingManager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listStudents"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="labelGrade">
         <property name="text">
          <string>Grade for the Student:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditGrade"/>
       </item>
       <item>
        <widget class="QPushButton" name="buttonGrade">
         <property name="text">
          <string>Grade</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="gradingmanager.qrc"/>
 </resources>
 <connections/>
</ui>
